### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by deccer**
Add more detail to install `mono-develop`

### CODE REVIEW

**Comment by andreakarasho** [comment]
Push it to the dev branch please

File: README.md [hunk 1]
```diff
@@ -57,10 +57,90 @@ is installed:
...
+```
+
+> for Ubuntu 18.10 - cosmic
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-cosmic main'
+```
+
+> for Ubuntu 18.04 - bionic
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-bionic main'
```
Comment (binff04) [inline README.md]: You can use a single command here:
```
sudo apt-add-repository "deb https://download.mono-project.com/repo/ubuntu stable-`lsb_release -cs` main"
```

Just to make sure `lsb_release` is installed, add this package `lsb-release` to the command where you install other required packages.

Comment (deccer) [reply README.md]: completely forgot about that, thanks

**Comment by andreakarasho** [comment]
i cant merge due to the presence of some conflicts

**Comment by deccer** [comment]
Oui, still busy with it. will mark it as a draft for now.

**Comment by andreakarasho** [comment]
With https://github.com/andreakarasho/ClassicUO/pull/1243 mono is not required anymore right? The dotnet CLI and SDK should be enough

**Comment by deccer** [comment]
Mono is still required, until we switch to dotnet core. These changes just calmed the compiler output down

**Pull request merged**